apps/condo/domains/analytics/schema/ExternalReport.js

```diff
@@ -87,19 +96,31 @@ const ExternalReport = new GQLListSchema('ExternalReport', {
             schemaDoc: 'Url for iframe',
             type: Virtual,
             resolver: async (item, _, context) => {
-                let link = ''
-                const userOrganizationLinkId = getUserOrganizationLinkId(context)
+                let link = null
```

Comment:
the virtual field should not be trying to get value from filters. It may break the cache logic